tests/rt.sh

```diff
@@ -420,8 +420,8 @@ if [[ $CREATE_BASELINE == true ]]; then
   rsync -a "${RTPWD}"/FV3_* "${NEW_BASELINE}"/
   rsync -a "${RTPWD}"/WW3_* "${NEW_BASELINE}"/
 
-  # S2S baselines are only (so far) available on these machines
-  if [[ $MACHINE_ID = hera.* ]] || [[ $MACHINE_ID = orion.* ]] || [[ $MACHINE_ID = wcoss_dell_p3 ]]; then
+  # FIXME: S2S baselines are only available on these machines with Intel
+  if [[ $MACHINE_ID = hera.intel ]] || [[ $MACHINE_ID = orion.intel ]] || [[ $MACHINE_ID = wcoss_dell_p3 ]]; then
```

Comment:
This should fix https://github.com/ufs-community/ufs-weather-model/issues/233 I suppose. But it means we leave the 20201019 baseline broken for hera.gnu for ufs-weather-model hashes `764a38c5725feec60bf20f8aadc0046751968d1d` and `58e5a8cc2009999926b4def7c5f8f24e9292aaa7`? If so, that should be noted somewhere.

Reply:
Do we make note of every broken hash somewhere?
Once this is merged in, `develop` will not be broken.
The baselines don't need to be recreated as we are not testing GNU runs for S2S.
Or am I missing something?

Reply:
@climbfuji I made comments in #217 and #219 
Is that sufficient?

Reply:
Yes, thanks, that's good enough. We sometimes check out certain hashes way later (... happening right now when folks are going back to hashes from January to figure out differences in the results between GFSv16 prod and develop ...) and it is important to know if something was broken in a commit.

Reply:
> Do we make note of every broken hash somewhere?
> Once this is merged in, `develop` will not be broken.
> The baselines don't need to be recreated as we are not testing GNU runs for S2S.
> Or am I missing something?

I needed to create new baselines (because of the HWRF physics changes and additional regression tests and that didn't work after #217) in order to see if all tests run to completion and give the same results from run to run. That triggered the error.